webapp/routes/products.js

```diff
@@ -0,0 +1,23 @@
+import express from "express"
+import _ from "lodash"
```

Comment:
Please import only those methods you need in your code instaead of importing the whole library. Use
`import { find } from 'lodash'` instead

BTW there's a native implementation [Array.find()](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/find)